recipe/meta.yaml

```diff
@@ -36,10 +36,21 @@ requirements:
   host:
 
 test:
+  requires:
+    - {{ compiler('c') }}
+    - pocl-cpu
```

Comment:
This creates a cycle in dependency chain.

Reply:
I removed the dependency in [09e3daf](https://github.com/conda-forge/ocl-icd-feedstock/pull/27/commits/09e3dafaf66a5e27e4ed64effe1b2b36baca153e)